src/runtime/pkg/katautils/katatrace/tracing.go

```diff
@@ -163,8 +164,33 @@ func Trace(parent context.Context, logger *logrus.Entry, name string, tags ...ma
 	return span, ctx
 }
 
-// AddTag adds an additional key-value pair to a tracing span. This can be used to
+// AddTags adds additional key-value pairs to a tracing span. This can be used to
 // provide dynamic tags that are determined at runtime.
-func AddTag(span otelTrace.Span, key string, value interface{}) {
```

Comment:
Thanks, @cmaf , set multiple key-value pairs at once is really helpful.
But I hope to keep the `AddTag`, this will make users easy to set up only one tag without specifying `map[string]string` type definition.